app/models/supplejack_api/concerns/user_set.rb

```diff
@@ -95,6 +95,28 @@ def self.featured_sets(num = 16)
     def update_attributes_and_embedded(new_attributes = {}, user = nil)
       new_attributes = new_attributes.try(:symbolize_keys) || {}
 
+      update_set_items(new_attributes)
+      update_featured_set(new_attributes, user)
+
+      self.attributes = new_attributes
```

Comment:
yeah I tried it with just attributes first, and it didn't work.  There are about a million concerns being pulled in, which i think may be the reason for having to use self here.  It was not my first instinct.